connector-drawer/src/main/java/com/redhat/cloud/notifications/connector/drawer/DrawerPayloadBuilder.java

```diff
@@ -0,0 +1,82 @@
+package com.redhat.cloud.notifications.connector.drawer;
+
+import com.redhat.cloud.notifications.connector.drawer.constant.ExchangeProperty;
+import com.redhat.cloud.notifications.connector.drawer.model.DrawerEntry;
+import com.redhat.cloud.notifications.connector.drawer.model.DrawerEntryPayload;
+import com.redhat.cloud.notifications.connector.drawer.model.DrawerUser;
+import io.vertx.core.json.Json;
+import jakarta.enterprise.context.ApplicationScoped;
+import org.apache.camel.Exchange;
+import org.apache.camel.Message;
+import org.apache.camel.Processor;
+import org.apache.camel.util.json.JsonObject;
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Map;
+
+import static com.redhat.cloud.notifications.connector.ExchangeProperty.ORG_ID;
+import static com.redhat.cloud.notifications.connector.ExchangeProperty.OUTCOME;
+import static com.redhat.cloud.notifications.connector.ExchangeProperty.START_TIME;
+import static com.redhat.cloud.notifications.connector.ExchangeProperty.SUCCESSFUL;
+import static com.redhat.cloud.notifications.connector.ExchangeProperty.TARGET_URL;
+import static com.redhat.cloud.notifications.connector.ExchangeProperty.TYPE;
+import static java.time.ZoneOffset.UTC;
+
+@ApplicationScoped
+public class DrawerPayloadBuilder implements Processor {
+
+    public static final String CE_SPEC_VERSION = "1.0.2";
+    public static final String CE_TYPE = "com.redhat.console.notifications.drawer";
+
+    @Override
+    public void process(Exchange exchange) {
+        DrawerUser entry = exchange.getMessage().getBody(DrawerUser.class);
+        final DrawerEntryPayload entryPayloadModel = exchange.getProperty(ExchangeProperty.DRAWER_ENTRY_PAYLOAD, DrawerEntryPayload.class);
+        DrawerEntryPayload payloadToSend = new DrawerEntryPayload();
```

Comment:
Why do we need a new instance? Can't we reuse `entryPayloadModel`?

Reply:
It was to sepatate the common data from the payloadModel we received from kafka, and the payload we will send to the drawer topic after updating the Id (one per user).
But if it's not easier to understand, we can use the same instance.

Reply:
Sorry, I didn't realize this was a 1-n situation. Using separate instances made sense. Please use the approach you prefer (current one or previous one) :smile:

Reply:
I think we can let it as it is for now